### DISCUSSION TITLE
Alias theme colours in normal tailwind config?

### DISCUSSION THREAD

**Discussion by jameschetwood** [saadeghi/daisyui#2321] [Q&A] [answered]
I want to alias theme colours with a semantic name. Eg I'd like `cardBackground` which I can set to `primary`.

```
const mytheme = {
  primary: "blue",
  secondary: "#ef9fbc",
  accent: "#eeaf3a",
  neutral: "#291334",
  "base-100": "white",
  cardBackground: "red",
};

const config: Config = {
  theme: {
    extend: {
      colors: {
        cardBackground: mytheme.primary,
      },
    },
  },
  daisyui: {
    themes: [
      "light",
      "dark",
      {
        mytheme,
      },
    ],
  },
};


function Card(){
	return <div className="bg-cardBackground">I'm a card</div>
}
```

The above works but I'd like to just use a string not a reference to an object.

This also seems to works but is there a simpler way of writting it? 

```
  colors: {
     cardBackground: "hsl(var(--p))",
  },
```

**Comment by saadeghi** [comment, marked as answer]
> This also seems to works but is there a simpler way of writting it?
> 
> ```
>   colors: {
>      cardBackground: "hsl(var(--p))",
>   },
> ```

I think that's the simplest way. [This table](https://daisyui.com/docs/colors/) is a guide to all CSS variables.
